client/src/components/reviews/List.jsx

```diff
@@ -81,11 +82,14 @@ class List extends React.Component {
               ;
             })}
           </div>
-          <div className='reviewHelpful'>
-            <p>Helpful? <button onClick={this.increaseHelfpulness.bind(this)} className='reviewYes'>Yes</button>({this.props.review.helpfulness}) | <span>Report</span></p>
+          <div className='reviewRecommended'>
+            {this.props.review.recommend && (<p>&#10003; I recommend this product</p>)}
           </div>
           <div className='reviewResponse'>
-            review response | {this.props.review.response}
+            {this.props.review.response === null ? '' : this.props.review.response.length > 0 ? (<p>Response: <br></br>{this.props.review.response}</p>) : ''}
```

Comment:
you could also just use <br /> instead of <br><br/> , I'm not sure if it has a big impact on anything though